server/routes/handlers/getCaseSearchHandler.js

```diff
@@ -3,28 +3,34 @@ const trackEvent = require('../../utils/analytics')
 const { settings } = require('../../../config')
 const getCaseSearchHandler = ({ searchCases }, getCaseSearchType) => async (req, res) => {
   const term = req.query.term
-  const type = getCaseSearchType(term)
+  const { searchType: type, error } = getCaseSearchType(term)
   const { cookies } = req
   const page = req.query.page > 0 ? req.query.page : undefined
+
+  if (error) {
+    res.render('case-search', { searchError: error, term })
+    return
+  }
+
```

Comment:
Should we log the error?

Reply:
It is user error and users will see it, not sure if it helps in any diagnostic. Any will go ahead and add it.